lluelles-lexisnexis.csl

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="fr-CA">
   <info>
-    <title>LexisNexis Québec (Guide Lluelles modifié, French - Canada)</title>
-    <id>http://www.zotero.org/styles/lluelles-lexisnexis</id>
-    <link href="http://www.zotero.org/styles/lluelles-lexisnexis" rel="self"/>
+    <title>Guide des références pour la rédaction juridique 7e édition (Notes complètes) (Guide Lluelles, No Ibid, French - Canada)</title>
+    <id>http://www.zotero.org/styles/lluelles-noibid</id>
```

Comment:
Per our existing "no ibid" styles, please use "no Ibid." in the title (no capitalized "No", and a period behind "Ibid", since it's an abbreviation). For the style name, ID, and "self" link, please use "-no-ibid" instead of "-noibid".